app/src/components/market/sections/market_verify/option/kleros_curation.tsx

```diff
@@ -238,7 +238,7 @@ export const KlerosCuration: FC<Props> = (props: Props) => {
             successful challenge.
           </DescriptionText>
           <RightButtonWrapper>
-            <RightButton buttonType={ButtonType.secondary}>
```

Comment:
Looks like there's also an instance of `<RightButton buttonType={ButtonType.secondary}>` at line 291. Is it the correct button type for that button?

Reply:
changed it! good catch!